packages/zip-it-and-ship-it/package.json

```diff
@@ -52,10 +52,9 @@
     "es-module-lexer": "^1.0.0",
     "esbuild": "0.25.4",
     "execa": "^8.0.0",
-    "fast-glob": "^3.3.2",
+    "fast-glob": "^3.3.3",
     "filter-obj": "^6.0.0",
     "find-up": "^7.0.0",
-    "glob": "^8.0.3",
```

Comment:
For anyone else reviewing this:
- glob@8 docs: https://github.com/isaacs/node-glob/tree/1b6bf20239a4c3bd73cacc82daa86bb7cf409398?tab=readme-ov-file#glob-primer
- fast-glob@3 docs: https://github.com/mrmlnc/fast-glob?tab=readme-ov-file#pattern-syntax